src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure implements IFigure {
+
+    protected double area;
+    protected Color color;
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public double getArea() {
+        return area;
+    }
+
+    public abstract void setArea();
+
+    public abstract void getInfo();
```

Comment:
previous comment is still not fixed, getInfo method should be in the interface, same with calculateArea(), let;s remove it from here

Reply:
fixed